### PR TITLE
Using the Particular.PlatformSample package

### PR DESCRIPTION
**Pull request by DavidBoike**
Waiting only for the final 1.0.0 package of Particular.PlatformSample, and a quick update to change package references from `1.0.0-*` to `1.*`.

### CODE REVIEW

File: samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitor/Program.cs [hunk 1]
```diff
@@ -13,12 +13,7 @@ static async Task Main()
         endpointConfiguration.UsePersistence<InMemoryPersistence>();
         endpointConfiguration.SendFailedMessagesTo("error");
 
-        var transport = endpointConfiguration.UseTransport<MsmqTransport>();
-        var routing = transport.Routing();
-        routing.RegisterPublisher(
-            typeof(ServiceControl.Contracts.MessageFailed).Assembly,
-            "Particular.ServiceControl"
-        );
+        var transport = endpointConfiguration.UseTransport<LearningTransport>();
```
Comment (danielmarbach) [inline samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitor/Program.cs]: Do we need the variable `transport` here or do we keep it around just for potential future use?

File: samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.StartupProjects.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 EndpointsMonitor\EndpointsMonitor.csproj
-NServiceBusEndpoint\NServiceBusEndpoint.csproj
\ No newline at end of file
+NServiceBusEndpoint\NServiceBusEndpoint.csproj
+PlatformLauncher\PlatformLauncher.csproj
```
Comment (danielmarbach) [inline samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.StartupProjects.txt]: In Monitoring, I called it Platform. Should we align on the name and also switch monitoring to  `PlatformLauncher`?

File: samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.Core.sln [hunk 3]
```diff
@@ -0,0 +1,30 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26730.12
+MinimumVisualStudioVersion = 15.0.26730.12
```
Comment (danielmarbach) [inline samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.Core.sln]: Run `projectStandards.linq` script

File: samples/servicecontrol/events-subscription/sample.md [hunk 4]
```diff
@@ -12,20 +12,21 @@ related:
...
+downloadbutton
+
 
 ## Running the project
 
+Running the project will result in 3 console windows:
+
+1. **NServiceBusEndpoint**: The endpoint that represents the system being monitored.
+1. **EndpointsMonitor**: The endpoint that subscribes to ServiceControl heartbeat and failed message events.
+1. **PlatformLauncher**: Runs an in-process version of ServiceControl and ServicePulse. When the ServiceControl instance is ready, a browser window will be launched displaying the ServicePulse Dashboard.
```
Comment (danielmarbach) [inline samples/servicecontrol/events-subscription/sample.md]: Is the dashboard the right entry point or should we point it to the failed message view?

File: samples/servicecontrol/monitoring3rdparty/sample.md [hunk 5]
```diff
@@ -10,23 +10,19 @@ redirects:
  - samples/custom-checks/monitoring3rdparty
 ---
 
-
-## Code walk-through
-
 When integrating with third-party systems, often messages fail when those systems are down. The Particular Platform has extensibility to enable the monitoring of third-party systems which are exposed as HTTP endpoints. This sample shows how to achieve this with custom checks.
 
+downloadbutton
```
Comment (danielmarbach) [inline samples/servicecontrol/monitoring3rdparty/sample.md]: Based on what convention do we use the downloadbutton? I thought we only add it to samples that we consider worth highlighting?

File: tools/generateNetCoreSamplesInclude.linq [hunk 6]
```diff
@@ -80,7 +80,7 @@ private void CreateYamlDeserializer()
     yamlDeserializer = builder.Build();
 }
 
-static Deserializer yamlDeserializer;
+static IDeserializer yamlDeserializer;
```
Comment (danielmarbach) [inline tools/generateNetCoreSamplesInclude.linq]: Why was this change necessary?

Comment (DavidBoike) [reply tools/generateNetCoreSamplesInclude.linq]: Because of a change YamlDotNet introduced. I've run into this too.

File: samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.StartupProjects.txt (hunk 2 above)
Comment (DavidBoike) [reply samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.StartupProjects.txt]: I really don't think it matters.

File: samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitor/Program.cs (hunk 1 above)
Comment (DavidBoike) [reply samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitor/Program.cs]: I don't mind it being there in samples so people don't get the idea it returns void.

File: samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.Core.sln (hunk 3 above)
Comment (DavidBoike) [reply samples/servicecontrol/events-subscription/ServiceControlContracts_1/EndpointsMonitoring.Core.sln]: Planning to run projectStandards on master once this is merged. There's lots of junk going on out there, far beyond this PR.

File: samples/servicecontrol/events-subscription/sample.md (hunk 4 above)
Comment (DavidBoike) [reply samples/servicecontrol/events-subscription/sample.md]: I played around with the failed messages view. It looks so boring before you have any failed messages that I decided the dashboard was the better way to go.
